docs/UsersGuide/source/BackgroundInfo/TechnicalOverview.rst

```diff
@@ -179,13 +179,17 @@ The ``ufs-srweather-app`` :term:`umbrella repository` is an NCO-compliant reposi
    │                ├── atmos_cubed_sphere
    │                └── ccpp
    ├── tests/WE2E
+   │     └── run_WE2E_tests.py 
    ├── ush
    │     ├── bash_utils
    │     ├── machine
-   │     ├── Python
-   │     ├── python_utils
-   │     ├── test_data
-   │     └── wrappers
+   │     ├── wrappers
+   │     ├── config.community.yaml
+   │     ├── config_defaults.yaml
+   │     ├── generate_FV3LAM_wflow.py
+   │     ├── launch_FV3LAM_wflow.sh
+   │     ├── setup.py
+   │     └── valid_param_vals.yaml
```

Comment:
I'll change the section title to "Repository Structure." I can see how maybe files wouldn't go in a "Directory Structure" diagram, but I think it's important to include files that users are likely to encounter when configuring their experiment or while looking into common error messages, so I wouldn't want to remove them entirely just because they don't quite fit the title. Maybe some of the files under `sorc` could be removed because they seem less relevant. Can't remember why I added them.